### ISSUE TITLE
BodyParameter issue with New-MgServicePrincipalClaimMappingPolicyByRef

### ISSUE THREAD

**Issue by ppanchal-mobiz** [microsoftgraph/msgraph-sdk-powershell#1461]
I am following the documentation provided at https://docs.microsoft.com/en-us/graph/api/serviceprincipal-post-claimsmappingpolicies?view=graph-rest-1.0&tabs=powershell

When trying to run the script given
```
Import-Module Microsoft.Graph.Applications

$params = @{
	"@odata.id" = "https://graph.microsoft.com/v1.0/policies/claimsMappingPolicies/cd3d9b57-0aee-4f25-8ee3-ac74ef5986a9"
}

New-MgServicePrincipalClaimMappingPolicyByRef -ServicePrincipalId $servicePrincipalId -BodyParameter $params
```

I get the following error:

> New-MgServicePrincipalClaimMappingPolicyByRef : A parameter cannot be found that matches parameter name
'BodyParameter'.

**Comment by robertreems** [comment]
TLDR: +1

One of my client pipelines failed after the https://www.powershellgallery.com/packages/Microsoft.Graph/ module has been upgraded to version 1.11.0. It was working without any issues using version 1.10.0.

I've looked for an alternative:
https://docs.microsoft.com/en-us/powershell/module/microsoft.graph.identity.directorymanagement/update-mgdirectoryadministrativeunit?view=graph-powershell-1.0

But as it turns out... I'm lacking some intelectual capabilities to understand the few words spend on documentation and examples. So I ended up calling the REST API directly with Invoke-RestMethod.

**Comment by peombwa** [comment]
Sorry about this, and thanks for surfacing the error with us. We've released a patch that fixes the issues at https://github.com/microsoftgraph/msgraph-sdk-powershell/releases/tag/1.11.1.

Please upgrade to the latest version of the module (`1.11.1`) using `Update-Module Microsoft.Graph`. Let me know if the patch addresses the issue on your end.

Related to #1440.

**Comment by ppanchal-mobiz** [comment]
Thank you, updating to 1.11.1 has solved the problem.
